docs/mcuboot_multiple_image_boot_flow.txt

```diff
@@ -0,0 +1,191 @@
+@startuml
+skinparam ParticipantPadding 60
+skinparam backgroundColor #FFFFF
+
+participant Main
+participant "Util/Misc"
+participant Loader
+participant Flash
+participant Verifier
+
+== Low level startup ==
+
+== Main ==
+
+note over Main: Device init:\n\
```

Comment:
This is a little odd to have lines ending with "\n\" in a text file. Can we just remove these?

Reply:
This is needed to have a good-looking rendered figure from the test description. I use plantUML plug-in Eclipse.